# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

geometry_msgs/Pose2D pose
float32[] poseErrorCovariance
float32 width
float32 widthErrorCovariance
uint32 filterID 
uint32 numberOfMeasurements
